avm.tflint.hcl

```diff
@@ -191,3 +191,69 @@ rule "role_assignments" {
 rule "tags" {
   enabled = true
 }
+
+# AVM WAF Rules
+
+rule "azurerm_application_gateway.zones" {
+  enabled = true
+}
+
+rule "azurerm_application_gateway.sku.name" {
+  enabled = true
+}
+
+rule "azurerm_cosmosdb_account.backup.type" {
+  enabled = true
+}
+
+rule "azurerm_kubernetes_cluster.zones" {
+  enabled = true
+}
+
+rule "azurerm_lb.sku" {
+  enabled = true
+}
+
+rule "azurerm_mysql_flexible_server.high_availability.mode" {
+  enabled = true
+}
+
+rule "azurerm_mysql_flexible_server.maintenance_window.day_of_week" {
+  enabled = true
+}
+
+rule "azurerm_postgresql_flexible_server.high_availability.mode" {
+  enabled = true
+}
+
+rule "azurerm_postgresql_flexible_server.maintenance_window.day_of_week" {
+  enabled = true
+}
+
+rule "azurerm_public_ip.sku" {
+  enabled = true
+}
+
+rule "azurerm_public_ip.zones" {
+  enabled = true
+}
+
+rule "azurerm_service_plan.zone_balancing_enabled" {
+  enabled = true
+}
+
+rule "azurerm_storage_account.account_replication_type" {
+  enabled = true
+}
+
+rule "azurerm_virtual_machine.zone" {
+  enabled = true
+}
+
+rule "azurerm_virtual_machine.zones" {
+  enabled = true
+}
+
+rule "azurerm_virtual_network_gateway.sku" {
+  enabled = true
+}
```

Comment:
We don't need them, you are correct. 

We just don't have any other method to easily identify the names of these rules. Putting them here makes it easier to make changes if you need to turn them off, etc. Admittedly that is an advanced use case and requires some overhead to maintain this. 

If we can automate the generation of these from the avm tflint plugin and add to the read me or similar, then this would not be needed. I can look into this alternative when I have some time.